### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by JoshuaSBrown**
1. Fix broken code, this was fixed in devel but needs to also be merged into main, the main loop of the agent was throwing an exception because the NATS nextMsg command was throwing a timeout. This is not an actual error, so the Timeout is caught and the loop is allowed to continue. 

2. Merge also includes work by @renan-souza for setting up docker compose for development.

3. Also includes badge to be added to main README.md indicating if all the CI tests are passing.

### CODE REVIEW

**Review by tskluzac — COMMENTED** [review]
left a print statement

File: zambeze/orchestration/queue/queue_nats.py [hunk 1]
```diff
@@ -133,8 +134,17 @@ async def nextMsg(self, channel: ChannelType) -> dict:
                 f"Cannot get next message client is not subscribed \
                         to any NATS topic: {channel.value}"
             )
-        msg = await self._sub[channel].next_msg(timeout=1)
-        data = json.loads(msg.data)
+
+        try:
+            msg = await self._sub[channel].next_msg(timeout=1)
```
Comment (tskluzac) [inline zambeze/orchestration/queue/queue_nats.py:139]: print here (not sure if we want this, or convert to a log / omit.

**Review by tskluzac — APPROVED** [review]
Looks great!

**Comment by JoshuaSBrown** [comment]
This PR cannot be merged until the race conditions in the CI integrations are merged in that are in pr https://github.com/ORNL/zambeze/pull/48.

**Review by renan-souza — APPROVED** [review]

**Comment by renan-souza** [comment]
@par-hermes format

**Comment by JoshuaSBrown** [comment]
Next time, I think we should create a branch off of devel and merge that into main, by using main it is a little disconcerting for reviewers that keep observing changes while they are looking through the merge request.

**Comment by JoshuaSBrown** [comment]
@wilkinson are we good to merge this?

**Pull request merged**